terminator/src/locator.rs

```diff
@@ -133,4 +133,8 @@ impl Locator {
             root: self.root.clone(),              // Inherit root
         }
     }
+
+    pub fn selector_string(&self) -> String {
+        format!("{:?}", self.selector)
```

Comment:
FYI It's needed since `self.selector` is private and need the Node.js bindings to access and convert to string.